tests/ui/empty_line_after_outer_attribute.rs

```diff
@@ -7,66 +7,70 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-
-
 #![warn(clippy::empty_line_after_outer_attr)]
 
 // This should produce a warning
 #[crate_type = "lib"]
 
 /// some comment
-fn with_one_newline_and_comment() { assert!(true) }
+fn with_one_newline_and_comment() {
+    assert!(true)
+}
 
 // This should not produce a warning
 #[crate_type = "lib"]
 /// some comment
-fn with_no_newline_and_comment() { assert!(true) }
-
+fn with_no_newline_and_comment() {
+    assert!(true)
+}
 
 // This should produce a warning
 #[crate_type = "lib"]
 
-fn with_one_newline() { assert!(true) }
+fn with_one_newline() {
+    assert!(true)
+}
 
 // This should produce a warning, too
 #[crate_type = "lib"]
 
-
-fn with_two_newlines() { assert!(true) }
```

Comment:
this needs to stay as before, as the lint is 'parsing' the span between the attribute and the following item. This test makes sure newlines are handled correctly.